ci/jenkins/jobs/projects-cloud.yaml

```diff
@@ -1656,3 +1656,40 @@
               default-excludes: true
               fingerprint: false
               only-if-success: false
+      - '{name}-{test_name}-for-pull-request':
+            test_name: kind-flexible-ipam-e2e
+            node: 'kind-flexible-ipam-testbed'
+            description: 'This is the {test_name} test for {name}.'
+            branches:
+            - ${{sha1}}      
+            builders:
+              - builder-kind-flexible-ipam-e2e:
+                  kind_cluster_name: '{test_name}'
+            trigger_phrase: ^(?!Thanks for your PR).*/test-kind-flexible-ipam-e2e.*
+            white_list_target_branches: []
+            allow_whitelist_orgs_as_admins: true
+            admin_list: '{antrea_admin_list}'
+            org_list: '{antrea_org_list}'
+            white_list: '{antrea_white_list}'
+            only_trigger_phrase: true
+            trigger_permit_all: true
+            status_context: jenkins-kind-flexible-ipam-e2e
+            status_url: --none--
```

Comment:
We can set the status URL to null, as it is now safe to be exposed.

Reply:
@KMAnju-2021 And could you also include the fix of https://github.com/antrea-io/antrea/pull/6802 in this PR? it's not urgent to update the job YAML since we've already created this new job in Jenkins.

Reply:
> @KMAnju-2021 And could you also include the fix of #6802 in this PR? it's not urgent to update the job YAML since we've already created this new job in Jenkins.

@KMAnju-2021 Did you address this comment?

Reply:
Hi @XinShuYang , the change in #6802 is not specific for GCP cloud VM. I think @KMAnju-2021 missed the changes in the past when the AntreaIPAM e2e on Kind is enabled, so there is no need to include it in this PR per my understanding.

Reply:
But I think we should merge this PR after the PR #6802 is merged.

Reply:
> But I think we should merge this PR after the PR #6802 is merged.

@luolanzone  Yes, we should fix the issue first. I thought the fix should be included in this PR because #6802 was already mentioned in the commit message.

Reply:
Hi @luolanzone  @XinShuYang ,Thanks for correcting me it was not specific to gcp ipam Testbed I missed the steps to add iptables rules for vlans in the test script, We should merge changes related to iptables rule first.